[fix #6] [mozmeao/infra#1104] Clean up oregon-b prod yaml.

jpetto opened the pull request:
- rename svc.yaml to web-svc.yaml

jpetto committed:
[fix #6] [mozmeao/infra#1104] Clean up oregon-b prod yaml.

- rename svc.yaml to web-svc.yaml

jgmize commented on oregon-b/basket-prod/clock-deploy.yaml:
```diff
@@ -48,381 +38,256 @@ spec:
-            secretKeyRef:
-              key: celery-num-workers
-              name: basket-prod-v50-env
+          value: 5
```
```suggestion
          value: "5"
```

jgmize commented on oregon-b/basket-prod/clock-deploy.yaml:
```diff
@@ -48,381 +38,256 @@ spec:
-            secretKeyRef:
-              key: celery-redis-max-connections
-              name: basket-prod-v50-env
+          value: 5
```
```suggestion
          value: "5"
```

jgmize commented on oregon-b/basket-prod/clock-deploy.yaml:
```diff
@@ -48,381 +38,256 @@ spec:
-            secretKeyRef:
-              key: email-port
-              name: basket-prod-v50-env
+          value: 587
```
```suggestion
          value: "587"
```

jgmize commented on oregon-b/basket-prod/clock-deploy.yaml:
```diff
@@ -48,381 +38,256 @@ spec:
-            secretKeyRef:
-              key: redis-cache-db
-              name: basket-prod-v50-env
+          value: 5
```
```suggestion
          value: "5"
```

jgmize commented on oregon-b/basket-prod/clock-deploy.yaml:
```diff
@@ -48,381 +38,256 @@ spec:
-            secretKeyRef:
-              key: redis-celery-db
-              name: basket-prod-v50-env
+          value: 4
```
```suggestion
          value: "4"
```

jgmize commented on oregon-b/basket-prod/clock-deploy.yaml:
```diff
@@ -48,381 +38,256 @@ spec:
-            secretKeyRef:
-              key: workflow-release-summary
-              name: basket-prod-v50-env
+          value: 10
```
```suggestion
          value: "10"
```

jgmize commented on oregon-b/basket-prod/clock-deploy.yaml:
```diff
@@ -48,381 +38,256 @@ spec:
-            secretKeyRef:
-              key: wsgi-num-workers
-              name: basket-prod-v50-env
+          value: 4
```
```suggestion
          value: "4"
```

jgmize commented on oregon-b/basket-prod/clock-deploy.yaml:
```diff
@@ -48,381 +38,256 @@ spec:
-              key: conflicts
-              name: basket-prod-v50-env
-        image: 127.0.0.1:5555/basket-prod:v50
+          value: 1
```
```suggestion
          value: "1"
```

jgmize commented on oregon-b/basket-prod/clock-deploy.yaml:
```diff
@@ -48,381 +38,256 @@ spec:
-              name: basket-prod-v50-env
-        image: 127.0.0.1:5555/basket-prod:v50
+          value: 1
+        image: mozmeao/basket:4d0668d
```
```suggestion
        image: mozmeao/basket:4d0668d4e5cb304e68b04bcc0d2b278d27b591e6
```

jgmize commented on oregon-b/basket-prod/donateworker-deploy.yaml:
```diff
@@ -48,381 +38,256 @@ spec:
-            secretKeyRef:
-              key: celery-num-workers
-              name: basket-prod-v50-env
+          value: 5
```
```suggestion
          value: "5"
```

jgmize commented on oregon-b/basket-prod/donateworker-deploy.yaml:
```diff
@@ -48,381 +38,256 @@ spec:
-            secretKeyRef:
-              key: celery-redis-max-connections
-              name: basket-prod-v50-env
+          value: 5
```
```suggestion
          value: "5"
```

jgmize commented on oregon-b/basket-prod/donateworker-deploy.yaml:
```diff
@@ -48,381 +38,256 @@ spec:
-            secretKeyRef:
-              key: email-port
-              name: basket-prod-v50-env
+          value: 587
```
```suggestion
          value: "587"
```

jgmize commented on oregon-b/basket-prod/donateworker-deploy.yaml:
```diff
@@ -48,381 +38,256 @@ spec:
-            secretKeyRef:
-              key: redis-cache-db
-              name: basket-prod-v50-env
+          value: 5
```
```suggestion
          value: "5"
```

jgmize commented on oregon-b/basket-prod/donateworker-deploy.yaml:
```diff
@@ -48,381 +38,256 @@ spec:
-            secretKeyRef:
-              key: redis-celery-db
-              name: basket-prod-v50-env
+          value: 4
```
```suggestion
          value: "4"
```

jgmize commented on oregon-b/basket-prod/donateworker-deploy.yaml:
```diff
@@ -48,381 +38,256 @@ spec:
-            secretKeyRef:
-              key: workflow-release-summary
-              name: basket-prod-v50-env
+          value: 10
```
```suggestion
          value: "10"
```

jgmize commented on oregon-b/basket-prod/donateworker-deploy.yaml:
```diff
@@ -48,381 +38,256 @@ spec:
-            secretKeyRef:
-              key: wsgi-num-workers
-              name: basket-prod-v50-env
+          value: 4
```
```suggestion
          value: "4"
```

jgmize commented on oregon-b/basket-prod/donateworker-deploy.yaml:
```diff
@@ -48,381 +38,256 @@ spec:
-              key: conflicts
-              name: basket-prod-v50-env
-        image: 127.0.0.1:5555/basket-prod:v50
+          value: 1
```
```suggestion
          value: "1"
```

jgmize commented on oregon-b/basket-prod/donateworker-deploy.yaml:
```diff
@@ -48,381 +38,256 @@ spec:
-              name: basket-prod-v50-env
-        image: 127.0.0.1:5555/basket-prod:v50
+          value: 1
+        image: mozmeao/basket:4d0668d
```
```suggestion
        image: mozmeao/basket:4d0668d4e5cb304e68b04bcc0d2b278d27b591e6
```

jgmize commented on oregon-b/basket-prod/fxaeventsworker-deploy.yaml:
```diff
@@ -48,381 +37,256 @@ spec:
-            secretKeyRef:
-              key: celery-num-workers
-              name: basket-prod-v50-env
+          value: 5
```
```suggestion
          value: "5"
```

jgmize commented on oregon-b/basket-prod/fxaeventsworker-deploy.yaml:
```diff
@@ -48,381 +37,256 @@ spec:
-            secretKeyRef:
-              key: celery-redis-max-connections
-              name: basket-prod-v50-env
+          value: 5
```
```suggestion
          value: "5"
```

jgmize commented on oregon-b/basket-prod/fxaeventsworker-deploy.yaml:
```diff
@@ -48,381 +37,256 @@ spec:
-            secretKeyRef:
-              key: email-port
-              name: basket-prod-v50-env
+          value: 587
```
```suggestion
          value: "587"
```

jgmize commented on oregon-b/basket-prod/fxaeventsworker-deploy.yaml:
```diff
@@ -48,381 +37,256 @@ spec:
-            secretKeyRef:
-              key: redis-cache-db
-              name: basket-prod-v50-env
+          value: 5
```
```suggestion
          value: "5"
```

jgmize commented on oregon-b/basket-prod/fxaeventsworker-deploy.yaml:
```diff
@@ -48,381 +37,256 @@ spec:
-            secretKeyRef:
-              key: redis-celery-db
-              name: basket-prod-v50-env
+          value: 4
```
```suggestion
          value: "4"
```

jgmize commented on oregon-b/basket-prod/fxaeventsworker-deploy.yaml:
```diff
@@ -48,381 +37,256 @@ spec:
-            secretKeyRef:
-              key: workflow-release-summary
-              name: basket-prod-v50-env
+          value: 10
```
```suggestion
          value: "10"
```

jgmize commented on oregon-b/basket-prod/fxaeventsworker-deploy.yaml:
```diff
@@ -48,381 +37,256 @@ spec:
-            secretKeyRef:
-              key: wsgi-num-workers
-              name: basket-prod-v50-env
+          value: 4
```
```suggestion
          value: "4"
```

jgmize commented on oregon-b/basket-prod/fxaeventsworker-deploy.yaml:
```diff
@@ -48,381 +37,256 @@ spec:
-              key: conflicts
-              name: basket-prod-v50-env
-        image: 127.0.0.1:5555/basket-prod:v50
+          value: 1
```
```suggestion
          value: "1"
```

jgmize commented on oregon-b/basket-prod/fxaeventsworker-deploy.yaml:
```diff
@@ -48,381 +37,256 @@ spec:
-              name: basket-prod-v50-env
-        image: 127.0.0.1:5555/basket-prod:v50
+          value: 1
+        image: mozmeao/basket:4d0668d
```
```suggestion
        image: mozmeao/basket:4d0668d4e5cb304e68b04bcc0d2b278d27b591e6
```

jgmize commented on oregon-b/basket-prod/fxaworker-deploy.yaml:
```diff
@@ -48,381 +37,256 @@ spec:
-            secretKeyRef:
-              key: celery-num-workers
-              name: basket-prod-v50-env
+          value: 5
```
```suggestion
          value: "5"
```

jgmize commented on oregon-b/basket-prod/fxaworker-deploy.yaml:
```diff
@@ -48,381 +37,256 @@ spec:
-            secretKeyRef:
-              key: celery-redis-max-connections
-              name: basket-prod-v50-env
+          value: 5
```
```suggestion
          value: "5"
```

jgmize commented on oregon-b/basket-prod/fxaworker-deploy.yaml:
```diff
@@ -48,381 +37,256 @@ spec:
-            secretKeyRef:
-              key: email-port
-              name: basket-prod-v50-env
+          value: 587
```
```suggestion
          value: "587"
```

jgmize commented on oregon-b/basket-prod/fxaworker-deploy.yaml:
```diff
@@ -48,381 +37,256 @@ spec:
-            secretKeyRef:
-              key: redis-cache-db
-              name: basket-prod-v50-env
+          value: 5
```
```suggestion
          value: "5"
```

jgmize commented on oregon-b/basket-prod/fxaworker-deploy.yaml:
```diff
@@ -48,381 +37,256 @@ spec:
-            secretKeyRef:
-              key: redis-celery-db
-              name: basket-prod-v50-env
+          value: 4
```
```suggestion
          value: "4"
```

jgmize commented on oregon-b/basket-prod/fxaworker-deploy.yaml:
```diff
@@ -48,381 +37,256 @@ spec:
-            secretKeyRef:
-              key: workflow-release-summary
-              name: basket-prod-v50-env
+          value: 10
```
```suggestion
          value: "10"
```

jgmize commented on oregon-b/basket-prod/ns.yaml:
```diff
@@ -1,13 +1,7 @@
-  resourceVersion: "1120259"
-  selfLink: /api/v1/namespaces/basket-prod
-  uid: 71ae6221-0152-11e8-8c8b-0656fabf84c0
 spec:
```
Please remove the spec & status sections

jgmize commented on oregon-b/basket-prod/web-deploy.yaml:
```diff
@@ -48,381 +37,256 @@ spec:
-            secretKeyRef:
-              key: email-port
-              name: basket-prod-v50-env
+          value: 587
```
```suggestion
          value: "587"
```

jgmize commented on oregon-b/basket-prod/web-deploy.yaml:
```diff
@@ -48,381 +37,256 @@ spec:
-            secretKeyRef:
-              key: celery-num-workers
-              name: basket-prod-v50-env
+          value: 5
```
```suggestion
          value: "5"
```

jgmize commented on oregon-b/basket-prod/web-deploy.yaml:
```diff
@@ -48,381 +37,256 @@ spec:
-            secretKeyRef:
-              key: celery-redis-max-connections
-              name: basket-prod-v50-env
+          value: 5
```
```suggestion
          value: "5"
```

jgmize commented on oregon-b/basket-prod/web-deploy.yaml:
```diff
@@ -48,381 +37,256 @@ spec:
-            secretKeyRef:
-              key: redis-cache-db
-              name: basket-prod-v50-env
+          value: 5
```
```suggestion
          value: "5"
```

jgmize commented on oregon-b/basket-prod/web-deploy.yaml:
```diff
@@ -48,381 +37,256 @@ spec:
-            secretKeyRef:
-              key: redis-celery-db
-              name: basket-prod-v50-env
+          value: 4
```
```suggestion
          value: "4"
```

jgmize commented on oregon-b/basket-prod/web-deploy.yaml:
```diff
@@ -48,381 +37,256 @@ spec:
-            secretKeyRef:
-              key: workflow-release-summary
-              name: basket-prod-v50-env
+          value: 10
```
```suggestion
          value: "10"
```

jgmize commented on oregon-b/basket-prod/web-deploy.yaml:
```diff
@@ -48,381 +37,256 @@ spec:
-            secretKeyRef:
-              key: wsgi-num-workers
-              name: basket-prod-v50-env
+          value: 4
```
```suggestion
          value: "4"
```

jgmize commented on oregon-b/basket-prod/web-deploy.yaml:
```diff
@@ -48,381 +37,256 @@ spec:
-              key: conflicts
-              name: basket-prod-v50-env
-        image: 127.0.0.1:5555/basket-prod:v50
+          value: 1
```
```suggestion
          value: "1"
```

jgmize commented on oregon-b/basket-prod/web-deploy.yaml:
```diff
@@ -48,381 +37,256 @@ spec:
-              name: basket-prod-v50-env
-        image: 127.0.0.1:5555/basket-prod:v50
+          value: 1
+        image: mozmeao/basket:4d0668d
```
```suggestion
        image: mozmeao/basket:4d0668d4e5cb304e68b04bcc0d2b278d27b591e6
```

jgmize commented on oregon-b/basket-prod/web-svc.yaml:
```diff
@@ -0,0 +1,23 @@
+    type: web
+  sessionAffinity: None
+  type: NodePort
+status:
```
Please remove the status section

jgmize commented on oregon-b/basket-prod/worker-deploy.yaml:
```diff
@@ -48,381 +37,256 @@ spec:
-            secretKeyRef:
-              key: celery-num-workers
-              name: basket-prod-v50-env
+          value: 5
```
```suggestion
          value: "5"
```

jgmize commented on oregon-b/basket-prod/worker-deploy.yaml:
```diff
@@ -48,381 +37,256 @@ spec:
-            secretKeyRef:
-              key: celery-redis-max-connections
-              name: basket-prod-v50-env
+          value: 5
```
```suggestion
          value: "5"
```

jgmize commented on oregon-b/basket-prod/worker-deploy.yaml:
```diff
@@ -48,381 +37,256 @@ spec:
-            secretKeyRef:
-              key: email-port
-              name: basket-prod-v50-env
+          value: 587
```
```suggestion
          value: "587"
```

jgmize commented on oregon-b/basket-prod/worker-deploy.yaml:
```diff
@@ -48,381 +37,256 @@ spec:
-            secretKeyRef:
-              key: redis-cache-db
-              name: basket-prod-v50-env
+          value: 5
```
```suggestion
          value: "5"
```

jgmize commented on oregon-b/basket-prod/worker-deploy.yaml:
```diff
@@ -48,381 +37,256 @@ spec:
-            secretKeyRef:
-              key: redis-celery-db
-              name: basket-prod-v50-env
+          value: 4
```
```suggestion
          value: "4"
```

jgmize commented on oregon-b/basket-prod/worker-deploy.yaml:
```diff
@@ -48,381 +37,256 @@ spec:
-            secretKeyRef:
-              key: workflow-release-summary
-              name: basket-prod-v50-env
+          value: 10
```
```suggestion
          value: "10"
```

jgmize commented on oregon-b/basket-prod/worker-deploy.yaml:
```diff
@@ -48,381 +37,256 @@ spec:
-            secretKeyRef:
-              key: wsgi-num-workers
-              name: basket-prod-v50-env
+          value: 4
```
```suggestion
          value: "4"
```

jgmize commented on oregon-b/basket-prod/worker-deploy.yaml:
```diff
@@ -48,381 +37,256 @@ spec:
-              key: conflicts
-              name: basket-prod-v50-env
-        image: 127.0.0.1:5555/basket-prod:v50
+          value: 1
```
```suggestion
          value: "1"
```

jgmize commented on oregon-b/basket-prod/worker-deploy.yaml:
```diff
@@ -48,381 +37,256 @@ spec:
-              name: basket-prod-v50-env
-        image: 127.0.0.1:5555/basket-prod:v50
+          value: 1
+        image: mozmeao/basket:4d0668d
```
```suggestion
        image: mozmeao/basket:4d0668d4e5cb304e68b04bcc0d2b278d27b591e6
```

jpetto committed:
Review fixes:

- remove superfluous data from ns.yaml
- quote all numeric env var values
- use full image hash

jpetto committed:
Further review fixes:

- remove status section from web-svc.yaml
- fix formatting issue in worker-deploy.yaml

jpetto commented:
@jgmize okay, this should be ready for final review. apologies for missing those few original review comments.

jgmize reviewed (approved):
Thanks @jpetto
